core/src/OpenMPTarget/Kokkos_OpenMPTarget_Parallel_MDRange.hpp

```diff
@@ -263,8 +282,12 @@ class ParallelFor<FunctorType, Kokkos::MDRangePolicy<Traits...>,
     for (ptrdiff_t i0 = begin_0; i0 < end_0; i0++)
       for (ptrdiff_t i1 = begin_1; i1 < end_1; i1++)
         for (ptrdiff_t i2 = begin_2; i2 < end_2; i2++)
-          for (ptrdiff_t i3 = begin_3; i3 < end_3; i3++)
-            functor(i0, i1, i2, i3);
+          for (ptrdiff_t i3 = begin_3; i3 < end_3; i3++) {
```

Comment:
I would prefer you being consistent with the curly brace.  Put everywhere or nowhere.